frontend/src/store/facilityBooking/action.ts

```diff
@@ -2,8 +2,6 @@ import { Dispatch, GetState } from '../types'
 import { ActionTypes, Booking, BookingStatus, Facility, FACILITY_ACTIONS, TimeBlock, TimeBlockType } from './types'
 import { ENDPOINTS, DOMAINS, get, del, DOMAIN_URL, put } from '../endpoints'
 import dayjs from 'dayjs'
-import { getBlockHr } from '../../components/FacilityBooking/BookingSection'
```

Comment:
Seems like we don't really use this function anywhere else, can we just remove it from `BookingSection`?